keyboards/spacey/spacey.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 vanilla
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "spacey.h"
```

Comment:
This lets you set a default behavior that can be overridden, and will work for the configurator. 

```suggestion
#include "spacey.h"

void encoder_update_kb(uint8_t index, bool clockwise){
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if(clockwise) {
            tap_code(KC_VOLD);
        } else {
            tap_code(KC_VOLU);
        }
    }
    return;
} 

```